wiki/Difficulties/osu!/Easy/en.md

```diff
@@ -9,34 +12,34 @@ And try to avoid these:
 
-You'll want to use [Distance Snapping](Distance_Snap) to allow equal distances over time. So, **adjust the distance snap multiplier so that hit circles only one beat apart don't overlap each other.** For a circle size of 4, a distance snap of 0.5 should work. New players often find hit circles that overlap each other like that hard to read. Most of the time, just having them next to each other without overlapping is good enough.
+You'll want to use [Distance Snapping](/wiki/Beatmap_Editor/Distance_Snap) to allow equal distances over time. So, **adjust the distance snap multiplier so that hit circles only one beat apart don't overlap each other.** For a circle size of 4, a distance snap of 0.5 should work. New players often find hit circles that overlap each other like that hard to read. Most of the time, just having them next to each other without overlapping is good enough.
 
 | This is easier to follow                                | than something like this                                 |
 |---------------------------------------------------------|----------------------------------------------------------|
 | ![Farther Aparts](Farther_Aparts.png "Farther Aparts")  | ![Less so](Less_so.png "Less so")                        |
 | Beats that are farther apart, better for newer players. | Beats closer together, better for more advanced players. |
 
-Set the [Beat Snap Divisor](Beat_Snap_Divisor) to 1/1, then select the circle and hit play and start placing circles in time to the music.
+Set the [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) to 1/1, then select the circle and hit play and start placing circles in time to the music.
 
-Now, your main rhythm here should be every other beat. Start with the [downbeat](Timing) and start off by placing a hit circle every other beat. However, you'll find that this is boring and puts you to sleep. This is why we go back and edit things to spruce things up a bit so that players don't totally die of boredom. You're going to want to do three things:
+Now, your main rhythm here should be every other beat. Start with the [downbeat](/wiki/Beatmap_Editor/Timing) and start off by placing a hit circle every other beat. However, you'll find that this is boring and puts you to sleep. This is why we go back and edit things to spruce things up a bit so that players don't totally die of boredom. You're going to want to do three things:
 
 Being on the beat
 -----------------
 
-Assume that a "." (period) is a beat without a [hit circle](Hit_Object), and an "O" ("o", not a zero) is a beat with a hit circle (Easy difficulties should be mapped to the beat for the most part, with a few exceptions). Here's a simple pattern you can try:
+Assume that a "." (period) is a beat without a [hit circle](/wiki/Hit_Objects), and an "O" ("o", not a zero) is a beat with a hit circle (Easy difficulties should be mapped to the beat for the most part, with a few exceptions). Here's a simple pattern you can try:
```

Comment:
might as well put code in that period and O
since the font for code is monospace, you could remove the note that it is an "o", not a zero